src/main/java/com/hierynomus/ntlm/functions/NtlmFunctions.java

```diff
@@ -182,4 +260,17 @@ private static Cipher getDESCipher(byte[] key) {
         }
 
     }
+
+    private static Cipher getRC4Cipher(byte[] key) {
+        try {
+            Cipher bc = Cipher.getInstance("RC4", "BC");
+            SecretKeySpec rc4 = new SecretKeySpec(setupKey(key), "RC4");
```

Comment:
Is the `setupKey` also necessary here? I seem to remember that was DES specific, but I might be mistaken.